components/storage/ServiceSelection.vue

```diff
@@ -10,60 +10,45 @@
           matchingServices[i] ? 'has-background-info' : 'has-background-light',
         ]"
       >
-        <button
-          class="button-nostyle"
-          type="button"
-          :disabled="!matchingServices[i]"
-          @click="change(i)"
-        >
-          <span class="icon is-size-2"
-            ><i
-              :class="[
-                'mdi',
-                selectedServices[i] ||
-                (selectedServices[i] === null && matchingServices[i])
-                  ? 'mdi-checkbox-marked'
-                  : matchingServices[i]
-                  ? 'mdi-checkbox-blank'
-                  : 'mdi-checkbox-blank-off',
-              ]"
-          /></span>
-        </button>
-        <p class="is-size-5 has-text-bold">{{ humanize(s.name) }}</p>
-        <span class="icon is-clickable" @click="toggleShowModal(s)"
-          ><i class="mdi mdi-information"
-        /></span>
+        <div>
+          <button
+            class="button-nostyle"
+            type="button"
+            :disabled="!matchingServices[i]"
+            @click="change(i)"
+          >
+            <span class="icon is-size-2"
+              ><i
+                :class="[
+                  'mdi',
+                  selectedServices[i] ||
+                  (selectedServices[i] === null && matchingServices[i])
+                    ? 'mdi-checkbox-marked'
+                    : matchingServices[i]
+                    ? 'mdi-checkbox-blank'
+                    : 'mdi-checkbox-blank-off',
+                ]"
+            /></span>
+          </button>
+          <p class="is-size-5 has-text-bold">{{ humanize(s.name) }}</p>
+        </div>
+        <details v-if="s.description" class="service-details mb-4">
```

Comment:
Let's add a little margin on the lefthand side as well ! 

```suggestion
        <details v-if="s.description" class="service-details mb-4 ml-4">
```
It's currently looking like this:
<img width="505" alt="Screenshot 2024-09-30 at 2 47 44 PM" src="https://github.com/user-attachments/assets/e4c5384c-5b5c-45d1-941d-0e579a72772c">